src/presign/participant.rs

```diff
@@ -329,69 +331,62 @@ impl PresignParticipant {
     fn gen_round_one_msgs<R: RngCore + CryptoRng>(
         &mut self,
         rng: &mut R,
-        message: &Message,
+        sid: Identifier,
```

Comment:
Note: Changing `message` -> `sid` in these `gen_...` functions wasn't strictly necessary for this PR, however it did make the development process easier (as it helped me catch where message info was being used where it shouldn't have been). Given that this change will be reflected in the upcoming presign audit, I think it makes sense to keep it here.

Reply:
This is fine by me!